tests/acceptance/features/loginUsingEmailAddress.feature

```diff
@@ -0,0 +1,41 @@
+@insulated @disablePreviews
+Feature: login users
+
+As an admin
+I want users to login with their email address stored on a LDAP server
+So that they only need to remember one username and password (SSO)
+
+	Scenario: login with default settings
+		When the LDAP users have been resyncted
+		Then it should be possible to login with the username "user1@example.org" and password "1234" into the WebUI
```

Comment:
Well, only if the login fiter contains the mail / mailPrimaryAddress / _insert ldap attribute containing the mail_

By default only `uid`/`cn` is compared. If uid/cn happens to contain an email, this will work. Otherwise it won't.